commands/import_test.go

```diff
@@ -114,10 +114,10 @@ func TestSetRolling(t *testing.T) {
 			exists, rolling, 1, 12, rolling, blank, blank, default12, cfg{0, rolling, 2, 12}, !returnsError},
 
 		tc{"rita import --rolling --chunk 0 --numchunks 24",
-			exists, rolling, 1, 12, rolling, 0, 24, default12, cfg{0, rolling, 0, 24}, returnsError},
+			exists, rolling, 1, 12, rolling, 0, 24, default12, cfg{0, rolling, 0, 24}, !returnsError},
```

Comment:
Updated the tests that increase the total chunks so that they no longer expect an error to be returned.